packages/storybook/src/spright/chat/message/chat-message.stories.ts

```diff
@@ -14,9 +14,22 @@ import { imgBlobUrl, markdownExample } from '../conversation/story-helpers';
 import { SpinnerAppearance } from '../../../../../nimble-components/src/spinner/types';
 import { ButtonAppearance } from '../../../../../nimble-components/src/menu-button/types';
 import { isChromatic } from '../../../utilities/isChromatic';
+import { iconThumbUpTag } from '../../../../../nimble-components/src/icons/thumb-up';
+import { iconThumbDownTag } from '../../../../../nimble-components/src/icons/thumb-down';
+
+const footerActionsDescription = `Place a button at the bottom of the message to allow the user to invoke a custom action.
+
+The buttons should have the \`ghost\` appearance and \`content-hidden\`.
+
+Nimble will set the height of the buttons to \`$ni-nimble-control-slim-height\`.
+`;
+
+const followUpPromptsDescription = 'Place buttons with followup prompts. They are below any action buttons.';
```

Comment:
Let's add the guidance about follow up button appearance and content here too.
```suggestion
const followUpPromptsDescription = 'Place 0 or more buttons with followup prompts. They appear below any action buttons.';
```

Reply:
Updated description as suggested.